src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(100));
```

Comment:
Think about which variables should be local in the method and which should be class-level

All magic numbers in your code should be constants.


see common mistakes https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery